html5/js/Client.js

```diff
@@ -1651,8 +1651,8 @@ class XpraClient {
       return true;
     }
 
-    if (this.server_readonly || !this.connected) {
-      return window == undefined;
+    if (this.server_readonly || !this.connected || (!win && this.server_is_shadow)) {
+      return win == undefined;
```

Comment:
This looks like a separate bug fix, is it not?

Reply:
Yeah, I saw that `window` wasn't defined while `win` was, so I assumed it's a typo